android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -271,6 +273,14 @@ public void showEditor()
       EditorActivity.start(this);
   }
 
+  public void showReportForm()
+  {
+    if (mIsFragmentContainer)
+      replaceFragment(ReportFragment.class, null, null);
```

Comment:
Спасибо что заметил.
> On Mar 15, 2016, at 8:41 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/MwmActivity.java <https://github.com/mapsme/omim/pull/2250#discussion_r56207326>:
> 
> > @@ -271,6 +273,14 @@ public void showEditor()
> >        EditorActivity.start(this);
> >    }
> >  
> > +  public void showReportForm()
> > +  {
> > +    if (mIsFragmentContainer)
> > +      replaceFragment(ReportFragment.class, null, null);
> Фрагмент должен в боковой панельке появиться?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2250/files#r56207326>